### ISSUE TITLE
Do not allow to create a project if name already exists

### ISSUE THREAD

**Issue by pombredanne** [nexB/scancode.io#470]
When I create a new project with a download URL, the project fails to create only after sometimes and presumably the download completed telling me that there is another project with the same name already there.

**Comment by tdruez** [comment]
Duplicate of https://github.com/nexB/scancode.io/issues/366
